src/Common/src/Abstractions/Expression/IServiceResolver.cs

```diff
@@ -2,12 +2,14 @@
 // The .NET Foundation licenses this file to you under the Apache 2.0 License.
 // See the LICENSE file in the project root for more information.
 
-using Steeltoe.Common.Expression;
+using System;
+using System.Collections.Generic;
+using System.Text;
 
-namespace Steeltoe.Messaging.RabbitMQ.Expressions
+namespace Steeltoe.Common.Expression
 {
     public interface IServiceResolver
     {
-        object Resolve(IEvaluationContext context, string serviceName);
+        object Resolve(IEvaluationContext context, string beanName);
```

Comment:
This doesn't appear to be Spring-specific, should it stay as `serviceName`?

Reply:
Ya ... it should stay serviceName